base/applications/mspaint/lang/it-IT.rc

```diff
@@ -110,15 +111,15 @@ BEGIN
         MENUITEM "&Copia\tCtrl+C", IDM_EDITCOPY
         MENUITEM "I&ncolla\tCtrl+V", IDM_EDITPASTE
         MENUITEM SEPARATOR
-        MENUITEM "Crop &selection", IDM_CROPSELECTION
+        MENUITEM "Ritaglia &selezione", IDM_CROPSELECTION
```

Comment:
"Ritaglia &selezione" collides with
MENUITEM "Copia &su...", IDM_EDITCOPYTO at line 118.
Best is to use the 2k3 accels to solve that.

Reply:
I can't find this menu item. Is the best thing to strip it?

Reply:
Do a selection in paint, then open the right-click-menu on that selection to see the menu.

Reply:
Nothing, it's still not present.
![image](https://github.com/user-attachments/assets/4ab836fe-1f29-4e0c-9d53-e1693fe1d577)
Modern paint has it. But i don't know why, i have paint in english even if my whole system is in italian. So I can't get the correct hotkey.
![image](https://github.com/user-attachments/assets/0e17e259-6c1b-49e6-acc8-56f0fedf2d16)

Reply:
> I can't find this menu item. Is the best thing to strip it?

Check the source code first: https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=IDM_CROPSELECTION (see the main.cpp file at the end).
It's in the Edit menu.